R/fd_fdiv.R

```diff
@@ -48,6 +48,9 @@ fd_fdiv <- function(traits, sp_com) {
 
   } else {
 
+    # We may override actual species names but we don't care because species
+    # identity doesn't matter when no species-site matrix is provided.
+    rownames(traits) <- paste0("sp", seq_len(nrow(traits)))
```

Comment:
```suggestion
    if (is.null(rownames(traits))) {
      rownames(traits) <- paste0("sp", seq_len(nrow(traits)))
   }
```

This could be a less brutal approach. I'm not sure what should be preferred.

Reply:
The only use of keeping the original rownames IMO is for debugging. It may be nice to know which species is causing issues then.